docs/operators/how-to/run.md

```diff
@@ -0,0 +1,137 @@
+# Running nwaku
+
+Nwaku binaries can be built and run on Linux and macOS.
+Windows support is experimental.
+
+```sh
+# Run with default configuration
+./build/wakunode2
+
+# See available command line options
+./build/wakunode2 --help
+```
+
+## Default configuration
+
+By default a nwaku node will:
+- generate a new private key and libp2p identities after every restart.
+See [this tutorial](./configure-key.md) if you want to generate and configure a persistent private key.
+- listen for incoming libp2p connections on the default TCP port (`60000`)
+- enable `relay` protocol
+- subscribe to the default pubsub topic, namely `/waku/2/default-waku/proto`
+- enable `store` protocol, but only as a client.
+This implies that the nwaku node will not persist any historical messages itself,
+but can query `store` service peers who do so.
+To configure `store` as a service node,
+see [this tutorial](./configure-store.md).
+
+> **Note:** The commonly used `filter` and `lightpush` protocols are _not_ enabled by default.
```

Comment:
```suggestion
> **Note:** The `filter` and `lightpush` protocols are _not_ enabled by default.
```
How common are they ? :)

Reply:
Hahahaha. Not common enough, if you ask me. Fair point. Have changed wording in [6cb52dc](https://github.com/status-im/nwaku/pull/963/commits/6cb52dcfb28d0b68a1564e7e127172633e056ab2)